portal/templates/redesign/help-and-support_new.html

```diff
@@ -34,29 +37,224 @@ <h1 class="banner__text--primary">Help and support</h1>
-        <p class="body-text"><strong>+ I want to sign up as a Teacher. Why am I asked for my 'professional' email address when registering?</strong></p>
-        <p class="body-text"><strong>+ What should I do when my students leave my school or club?</strong></p>
-        <p class="body-text"><strong>+ What is an independent student?</strong></p>
-        <p class="body-text"><strong>+ What if a student moves to another school? Can they keep their progress?</strong></p>
-        <p class="body-text"><strong>+ What do I need to do to allow an independent student to join my class or school?</strong></p>
-        <p class="body-text"><strong>+ I've set up my students. How do they log in?</strong></p>
-        <p class="body-text"><strong>+ As a student in a school or club, who can I share my custom created levels with?</strong></p>
-        <p class="body-text"><strong>+ As a teacher, who can I share my custom created levels with?</strong></p>
-        <p class="body-text"><strong>+ How can I control the levels my students create?</strong></p>
-        <p class="body-text"><strong>+ I am an independent student. Why can't I share my custom created levels?</strong></p>
+        <h1>FAQs</h1>
+
+<style>
+ .dropdown
+
+    input + label:before,
+    input + label:hover:before{
+        content: '+';
+        color: red;
+    }
+    
+    input:checked + label:before,
+    input:checked + label:hover:before{
+        content: 'X';
+        color: red;
+    }
+
+    input:checked + label answer{
+        display: block;
+    }
+
+
+    input:checked ~ answer{
+        display: block;
+    }
+
+
+    input, answer{
+        display: none;
+    }
+
+
+</style>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q1" type="checkbox" />
+    <label for="FAQ-q1">
+            <strong>I am a school student and have forgotten my password. How do I reset it?</strong>
+     </label>
+        <answer>
+            <p>Your teacher is the only person who can update or reset your password, so you should speak to them.</p>
+        </answer>
+</div>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q2" type="checkbox" />
+    <label for="FAQ-q2">
+            <strong>I am a teacher and have forgotten my password. How do I reset it?</strong>
+    </label>
+        <answer>
+            <p>Go to the <a href=\"{% url 'teach' %}\">Teach</a> page and select the forgotten password link at the bottom of the log in box to be sent a link to reset your password.</p>
+        </answer>
+</div>
+
+
+<div class="body-text dropdown">
+    <input id="FAQ-q3" type="checkbox" />
+    <label for="FAQ-q3">
+            <strong>I am an independent student and have forgotten my password. How do I reset it?</strong>
+    </label>
+    <answer>
+            <p>Go to the <a href="{% url 'play' %}">Play</a> page, go to the independent students' login box and select the forgotten password link at the bottom of the log in box to be sent a link to reset your password.</p>
```

Comment:
Apostrophes should be replaced with `&rsquo;` to give them their curly appearance.